docs/source/formats.md

```diff
@@ -105,7 +105,25 @@ Add `site1-f5-a-concat.cfg` to the configs folder with the rest of the devices.
 ```
 
 ### Palo Alto Networks
-Batfish supports Palo Alto Networks devices with or without Panorama. For each device, concatenate the following show commands into one file.
+Batfish supports Palo Alto Networks devices with or without Panorama.
+
+#### From Panorama (preferred)
+For devices managed through Panorama, with no configuration done directly on the device, you can pull configurations from Panorama without having to run any commands on the managed-devices themselves.
```

Comment:
What happens if Panorama is just managing part of the device config? Is there a process to merge the Panorama managed config with the one pulled from the device directly?